ui/app/pages/home/home.component.js

```diff
@@ -23,18 +23,21 @@ export default class Home extends PureComponent {
   }
 
   static defaultProps = {
-    unsetMigratedPrivacyMode: null,
     hasDaiV1Token: false,
+    activeTab: {},
   }
 
   static propTypes = {
+    activeTab: PropTypes.shape({
```

Comment:
This prop appears to be unused

Reply:
It would be nice if linting could catch this.

Reply:
It can! I used `react/no-unused-prop-types` to catch these last few. There are a lot of unused props throughout the codebase though, so I'm going to wait until after this PR lands before adding it to avoid churn.

Reply:
but thanks for catching these in the meantime :smile:

Reply:
ab0d66fa1